### PR TITLE
fisspy

### PR DESCRIPTION
**Pull request by jhkang0301**
Checklist

- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vend other packages
- [x] Build number is 0
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by jhkang0301** [comment]
@conda-forge/staged-recipes This is ready for review

File: recipes/fisspy/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+  sha256: b80ba11af1916e06e83d411909c6c5a7fa7e5060b2f6fa2cad2f90f6a6a29869
+
+build:
+  noarch: python
+  number: 1
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python>=3.6
```
Comment (scopatz) [inline recipes/fisspy/meta.yaml]: there needs to be a single space before the `>=`

**Comment by jhkang0301** [comment]
@scopatz Thank you for your comment. I fixed it.

File: recipes/fisspy/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: b80ba11af1916e06e83d411909c6c5a7fa7e5060b2f6fa2cad2f90f6a6a29869
+
+build:
+  noarch: python
+  number: 2
```
Comment (chrisburr) [inline recipes/fisspy/meta.yaml]: This should start at 0
```suggestion
  number: 0
```

File: recipes/fisspy/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+
+test:
+  imports:
+    - fisspy
+    - fisspy.read
+    - fisspy.analysis
+
+about:
+  home: http://fiss.snu.ac.kr/
+  license: BSD 2-Clause
```
Comment (chrisburr) [inline recipes/fisspy/meta.yaml]: The SPDX identifier has another dash:
```suggestion
  license: BSD-2-Clause
```

Comment (chrisburr) [reply recipes/fisspy/meta.yaml]: There also needs to be a licence file

**Comment by jhkang0301** [comment]
@chrisburr Thank you for your comments. I replaced the commented lines and upload the license file (LICENSE.txt).

**Comment by chrisburr** [comment]
Great, LGTM 👍 

Welcome to conda-forge! 🎉

**Pull request merged**